actions/local_storage.jsx

```diff
@@ -18,14 +18,3 @@ export function setPreviousTeamId(teamId) {
         return {data: true};
     };
 }
-
-export function setRecentEmojis(recentEmojis = []) {
```

Comment:
We dont need this anymore since we are using localstoragestore directly in the addRecentEmoji action